Example/Podfile

```diff
@@ -3,3 +3,11 @@ use_frameworks!
 target 'PinpointKitExample' do
 	pod 'PinpointKit', :path => '../'
 end
+
+post_install do |installer|
```

Comment:
Can we comment this section of the Podfile with links / explanation so future us know why we put it here?